source/features/conflict-marker.tsx

```diff
@@ -53,6 +56,7 @@ async function addIcon(links: HTMLAnchorElement[]): Promise<void> {
 async function init(signal: AbortSignal): Promise<void> {
 	await expectToken();
 	observe(openPrsListLink, batchedFunction(addIcon, {delay: 100}), {signal});
+	observe(PrsListInOpenIssuesLink, batchedFunction(addIcon, {delay: 100}), {signal});
```

Comment:
`PrsListInOpenIssuesLink` is the same as `openPrsListLink`, it shouldn't be a new selector/observer. Please add the new selector to `openPrsListLink`, which can be turned into an array, probably exactly this:

```js
export const openPrsListLink = [
	// `.color-fg-open` is needed because of the icon added by `highlight-non-default-base-branch`
	css`
		.js-issue-row:has(
			.octicon-git-pull-request.color-fg-open,
			.octicon-git-pull-request-draft
		) a.js-navigation-open
	`,
	// React view
	css`
		li[role="listitem"] a[data-hovercard-url*="/pull"]
	`,
];
```